addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -423,6 +423,7 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                 return event.data.onSuccess();
             case 'edit_menu':
                 return this.dialogs.add(EditMenuDialog, {
+                    ...params[0],
```

Comment:
@detrouxdev Yes it's clearer this way :+1: but only if you're not planning to add more values ! Otherwise, it will be better to spread `params[0]` into dialog props :+1:

Reply:
That's fair I suppose, thanks for adapting regardless!